packages/config/index.js

```diff
@@ -22,21 +22,24 @@ const getAppEntry = (rootFolder, isProd) => {
 };
 
 module.exports = (configurations) => {
+    const isProd = configurations.isProd || process.env.NODE_ENV === 'production';
```

Comment:
Checking for production was inconsistently done across the config. I added a definitive check here.